central/reports/scheduler/schedule_test.go

```diff
@@ -0,0 +1,59 @@
+package scheduler
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/images/defaults"
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+var expectedVulnReportEmailTemplateRhacsBranding = `
+	Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+var expectedVulnReportEmailTemplateStackroxBranding = `
+	StackRox has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+var expectedNoVulnsFoundEmailTemplateRhacsBranding = `
+	Red Hat Advanced Cluster Security for Kubernetes has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+var expectedNoVulnsFoundEmailTemplateStackroxBranding = `
+	StackRox has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+func TestVulnMessageBranding(t *testing.T) {
```

Comment:
I suggest doing the change mentioned above (moving all templated strings to `text/template` rather than `fmt`) and change these tests to call `formatMessage` rather than these special methods for each template.

Our tests will be written from the following perspective:
- Given that env `X` is set 
- When I call `formatMessage`
- Then I get message `Y` back

Reply:
Reformatted tests like this already after Mishas feedback. Found a nice way to mock ReportConfiguration